include/spdlog/sinks/dup_filter_sink.h

```diff
@@ -41,15 +41,17 @@ class dup_filter_sink : public dist_sink<Mutex>
 {
 public:
     template<class Rep, class Period>
-    explicit dup_filter_sink(std::chrono::duration<Rep, Period> max_skip_duration)
+    explicit dup_filter_sink(std::chrono::duration<Rep, Period> max_skip_duration, level::level_enum level = level::info)
```

Comment:
Please provide more descriptive name to this param. something like `level::level_enum notification_level)`

Reply:
ok